client/generated_backing_image.go

```diff
@@ -11,9 +11,13 @@ type BackingImage struct {
 
 	DiskStateMap map[string]string `json:"diskStateMap,omitempty" yaml:"disk_state_map,omitempty"`
 
+	DownloadProgressMap map[string]string `json:"downloadProgressMap,omitempty" yaml:"download_progress_map,omitempty"`
```

Comment:
Do you change anything relatd to backing image?

Reply:
No, it got automatically updated from the generator script.